website/src/models/user.js

```diff
@@ -390,7 +390,16 @@ var UserSchema = new Schema({
       hatchPet: {type: Boolean, 'default': false},
       raisePet: {type: Boolean, 'default': false},
       streak: {type: Boolean, 'default': false}
-    }
+    },
+    improvementCategories: {type: Array, validate: function (categories) {
```

Comment:
Can we split this onto multiple lines? 

```
improvementCategories: {
  type: Array, 
  validate: function (categories) {
```

We could also use an arrow function since we don't care about `this`

```
validate: (categories) => {
```